Makefile

```
@@ -6,9 +6,9 @@ fix:
 	cargo fmt
 
 preflight:
+	cargo audit
```

This will fail faster (if it ever does)